molecule/tests/test_default.py

```diff
@@ -52,6 +52,23 @@ def test_files_exist(host, f):
     assert host.file(f).exists
 
 
+@pytest.mark.parametrize(
+    "d,count",
+    [
+        # This directory should contain _only_ cisa_cool_retro_0.png.
```

Comment:
It's pedantic but all this guarantees is that there is only one file in the directory.

Reply:
It does, but when coupled with this test, it guarantees that one file is the one we want:
https://github.com/cisagov/ansible-role-xfce-cool/blob/a977f3f7d7f08b85062a449eeecde95f48300105/molecule/tests/test_default.py#L41-L52

Reply:
Ah, right, right. Thanks for the reminder. The comment just threw me off.